packages/oasgraph/src/resolver_builder.ts

```diff
@@ -510,7 +552,7 @@ function resolveLinkParameter(paramName: string, value: string, resolveData: any
     if (value === '$request.body') {
       return resolveData.usedPayload
 
-    // CASE: parameter in previous body
+      // CASE: parameter in previous body
```

Comment:
I believe in our style convention, we usually keep comments indented the same as the surrounding code. Would it be possible to move these back?

Reply:
I will change my vscode linter rules for this project. Hope it is possible, if not I will install jslint.

Reply:
Ok it's done and will be shipped in the next commit.